src/components/Template/Template.test.tsx

```diff
@@ -1,239 +1,60 @@
 import { axe } from "jest-axe";
-import * as React from "react";
 import renderer from "react-test-renderer";
 
 import {
   Template,
-  TemplateAboveHeader,
-  TemplateHeader,
   TemplateBreakout,
-  TemplateContent,
-  TemplateContentTop,
-  TemplateContentPrimary,
-  TemplateContentSidebar,
-  TemplateFooter,
-  TemplateAppContainer,
+  TemplateTop,
+  TemplateMain,
+  TemplateSidebar,
+  TemplateBottom,
 } from "./Template";
 import Placeholder from "../Placeholder/Placeholder";
 
-const aboveHeader = <Placeholder variant="short">Above Header</Placeholder>;
-const header = <Placeholder variant="short">NYPL Header</Placeholder>;
-const breakout = (
-  <>
-    <Placeholder variant="short">Breadcrumbs</Placeholder>
-    <Placeholder variant="short">Hero</Placeholder>
-  </>
-);
+const breakout = <Placeholder variant="short">Breakout</Placeholder>;
 const sidebar = "left";
 const contentTop = <Placeholder>Content Top</Placeholder>;
 const contentSidebar = <Placeholder>Left Sidebar</Placeholder>;
-const contentPrimary = (
+const contentMain = (
   <>
     <Placeholder>Main Content</Placeholder>
     <Placeholder variant="short">More Content</Placeholder>
   </>
 );
-const footer = <Placeholder variant="short">Footer</Placeholder>;
-
-describe("TemplateAppContainer accessibility", () => {
-  it("passes axe accessibility test", async () => {
-    const { container } = render(
-      <TemplateAppContainer
-        aboveHeader={aboveHeader}
-        header={header}
-        breakout={breakout}
-        sidebar={sidebar}
-        contentTop={contentTop}
-        contentSidebar={contentSidebar}
-        contentPrimary={contentPrimary}
-        footer={footer}
-      />
-    );
-    expect(await axe(container)).toHaveNoViolations();
-  });
-});
+const contentBottom = <Placeholder variant="short">Content Bottom</Placeholder>;
+
+const templateChildren = (
+  <Template sidebar={sidebar}>
+    <TemplateBreakout>{breakout}</TemplateBreakout>
+    <TemplateTop>{contentTop}</TemplateTop>
+    <TemplateSidebar>{contentSidebar}</TemplateSidebar>
+    <TemplateMain>{contentMain}</TemplateMain>
+    <TemplateBottom>{contentBottom}</TemplateBottom>
+  </Template>
+);
 
 describe("Template components accessibility", () => {
   it("passes axe accessibility test", async () => {
-    const { container } = render(
-      <Template>
-        <TemplateAboveHeader>{aboveHeader}</TemplateAboveHeader>
-        <TemplateHeader>
-          {header}
-          <TemplateBreakout>{breakout}</TemplateBreakout>
-        </TemplateHeader>
-        <TemplateContent sidebar={sidebar}>
-          <TemplateContentTop>{contentTop}</TemplateContentTop>
-          <TemplateContentSidebar>{contentSidebar}</TemplateContentSidebar>
-          <TemplateContentPrimary>{contentPrimary}</TemplateContentPrimary>
-        </TemplateContent>
-        <TemplateFooter>{footer}</TemplateFooter>
-      </Template>
-    );
+    const { container } = render(<>{templateChildren}</>);
```

Comment:
I think it'd be good to make `templateChildren` a function and pass in the value of the sidebar, either left or right. Then you can have two accessibility tests and verify both sidebar variants (and the default without a sidebar) are all accessible.